circle.yml

```diff
@@ -2,32 +2,14 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/python:2.7.14-browsers
+      - image: themattrix/tox
```

Comment:
@farhaanbukhsh while this does simplify running the tests locally, I'm not sure this image is compatible with the CircleCI environment. CircleCI 2.0 supports custom images but it seems there are specific needs for that. [ref](https://circleci.com/docs/2.0/custom-images/)

I recommend using a Python3 image from CircleCI such as https://circleci.com/developer/images/image/cimg/python pinned to v3.5 as that's in use by the platform.